packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js

```diff
@@ -152,7 +152,8 @@ PrayerCard.defaultProps = {
   showHelp: true,
   header: true,
   avatarSize: 'small',
-  name: 'request',
+  name: 'Request',
+  source: 'NewSpring Church',
```

Comment:
So if the user doesn't have a campus it will default to NewSpring Church?

Reply:
Or we can just leave it blank. You choose.